### PR TITLE
Cut to Candidate (10/14)

### PR DESCRIPTION
**Pull request by ghost**
Cut to candidate with:

updated tag colors for aave
fixed label on staking page ('updated weekly' to 'updated daily), fix
fixed skeleton bg on C & E page
Fix symbol in modal and typo
Update README.md (#295) …
Worked on the skeleton cleanup on dark mode (#302)
Wored on the approval issue on the staking page (#293)
Worked on the nft btn dark mode issue (#301)
added BLP (Banker Joe) icons, colors, filter option (#306)
Use wallet provider/fallback to private node (#308) 
added retry at getting balance (#304)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,84 @@
...
 
 ## Summary
 
-This project is the App Page for Snowball and  a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+This project is the App Page for Snowball (front-end) and a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-### Local http://localhost:3000
-### Development https://jupiter-frontend.vercel.app/
+### Local: http://localhost:3000
+### Development: https://jupiter-frontend.vercel.app/
```
Comment (prodvdc) [inline README.md:13]: @Jonasslv Minor issue. Are we planning to change this URL?

**Review by prodvdc — APPROVED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:13]: yeah sure, we can take a look at this later @Ncookiez 
The current development URL is https://app-v2-mocha.vercel.app/

Comment (Ncookiez) [reply README.md:13]: Fixed it on #313 .

**Pull request merged**